tests/db/findMany.tests.ts

```diff
@@ -0,0 +1,25 @@
+import dotenv from 'dotenv';
+import path from 'path';
+import { expect } from 'chai';
+import { findMany } from '../../src/modules/database';
+import { FindRequest } from '../../src/types/codemash.dtos';
+
+describe('findMany', () => {
+	beforeEach(() => {
+		dotenv.config({
+			path: path.resolve(__dirname, '../data/config/.env'),
+		});
+	});
+
+	it('should get all records', async () => {
+		const request = new FindRequest({ collectionName: 'employees' });
+		const response = await findMany(request);
+		expect(response).to.be.not.null.and.to.have.key('isSuccess');
```

Comment:
isSuccess - ar mes turime toki key ?

Kiek zinau turime totalCount, schema, result, ... https://api.codemash.io/json/metadata?op=FindRequest

Siaip jei uzwrapinai actiona ir grazini isSuccess, isError - tai blogai. 

REST kodai nusako, cia gerai blogai, plius kai blogai ServiceStack API grazina info 

```json
{"responseStatus":{"errorCode":"String","message":"String","stackTrace":"String","errors":[{"errorCode":"String","fieldName":"String","message":"String","meta":{"String":"String"}}],"meta":{"String":"String"}}}
```
kazkas tokio

Reply:
cia kaip ir kalbejom, pareina is @servicestack/client libo apdorotas response ir prie jo visada buna isSuccess, isError